parser/src/main/kotlin/com/linkedin/dex/parser/FormatUtils.kt

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.dex.parser
+
+import com.linkedin.dex.spec.ClassDefItem
+import com.linkedin.dex.spec.DexFile
+
+/**
+ * Format a class descriptor in a human-readable format that is also compatible with Android instrumentation tests
+ *
+ * For example,
+ * "Lorg/junit/Test;"
+ * would become:
+ * "org.junit.Test"
+ */
+fun formatDescriptor(descriptor: String): String {
+    return descriptor
+            // strip off the "L" prefix
+            .substring(1)
+            // swap out slashes for periods
+            .replace('/', '.')
+            // strip off the ";"
+            .dropLast(1)
+}
+
+/**
+ * Extract the fully qualified class name from a [ClassDefItem] and format it for use with Android instrumentation tests
+ *
+ * @see [formatDescriptor]
+ */
+fun DexFile.formatClassName(classDefItem: ClassDefItem): String {
```

Comment:
Not sure the naming here makes sense to me... it doesn't so much format it for use with instrumentation tests as format it to be combined with a test name for instrumentation tests. What do you think about naming it more specifically? Not a huge deal